modules/yieldloveBidAdapter.js

```diff
@@ -0,0 +1,149 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import * as utils from '../src/utils.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const ENDPOINT_URL = '//s2s.yieldlove-ad-serving.net/openrtb2/auction';
+
+const DEFAULT_BID_TTL = 300; /* 5 minutes */
+const DEFAULT_CURRENCY = 'EUR';
+
+const participatedBidders = []
+
+export const spec = {
+  gvlid: 251,
+  code: 'yieldlove',
+  aliases: ['yl'],
```

Comment:
This alias is a duplicate of an existing adapter and should be removed or updated